config/prod/iAtlas-large.yaml

```diff
@@ -0,0 +1,48 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-ubuntu-v1.yaml
+stack_name: iAtlas-large
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "iAtlas"
+  # The resource owner
+  OwnerEmail: "andrew.lamb@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: {{stack_group_config.default_vpc}}
+  KeyName: {{stack_group_config.default_key_pair}}
+  AMIId: {{stack_group_config.default_ubuntu_ami}}
+
+  # Settings have default values but can be overriden. Set the below
+  # parameters *only* if you want to override the defaults.
+
+  # (Optional) EC2 instance type, default is "t3.nano" (other available types https://aws.amazon.com/ec2/instance-types/)
+  InstanceType: "r5.12xlarge"
```

Comment:
Also this instance isn't supported. @zaro0508 can you add support?

Reply:
Is there a list of supported instances?

Reply:
https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/managed-ec2-ubuntu-v1.yaml#L15

Reply:
changed to r4.16xlarge